rct229/rules/section22/section22rule2.py

```diff
@@ -0,0 +1,107 @@
+from rct229.rule_engine.rule_base import RuleDefinitionBase
+from rct229.rule_engine.rule_list_indexed_base import RuleDefinitionListIndexedBase
+from rct229.rule_engine.user_baseline_proposed_vals import UserBaselineProposedVals
+from rct229.schema.config import ureg
+from rct229.utils.assertions import getattr_
+from rct229.utils.jsonpath_utils import find_all
+from rct229.utils.std_comparisons import std_equal
+
+APPLICABLE_SYS_TYPES = [
+    "SYS-7",
+    "SYS-8",
+    "SYS-11.1",
+    "SYS-11.2",
+    "SYS-12",
+    "SYS-13",
+    "SYS-1A",
+    "SYS-3A",
+    "SYS-7A",
+    "SYS-8A",
+    "SYS-11.1A",
+    "SYS-11.2A",
+    "SYS-12A",
+    "SYS-13A",
+    "SYS-7B",
+    "SYS-8B",
+    "SYS-11B",
+    "SYS-12B",
+    "SYS-13B",
+    "SYS-1C",
+    "SYS-3C",
+    "SYS-7C",
+    "SYS-11C",
+    "SYS-12C",
+    "SYS-13C",
+]
+DESIGN_RETURN_TEMP = 56 * ureg("degF")
+
+
+class Section22Rule2(RuleDefinitionListIndexedBase):
+    """Rule 2 of ASHRAE 90.1-2019 Appendix G Section 22 (Hot water loop)"""
+
+    def __init__(self):
+        super(Section22Rule2, self).__init__(
+            rmrs_used=UserBaselineProposedVals(False, True, False),
+            each_rule=Section22Rule2.ChillerFluidLoopRule(),
+            index_rmr="baseline",
+            id="22-2",
+            description="Baseline chilled water design return temperature shall be modeled at 56F.",
+            rmr_context="ruleset_model_instances/0",
+            list_path="fluid_loops[*]",
+        )
+
+    def create_data(self, context, data):
+        rmi_b = context.baseline
+        chillers = find_all("$.chillers[*]", rmi_b)
+        loop_chiller_dict = {}
```

Comment:
As with several other rules, the RDS only uses the keys from this dictionary. So it would be more natural to collect a list `chiller_loop_ids` instead of a dict.